features/pointer-events.yml

```
@@ -0,0 +1,6 @@
+name: pointer-events
+description: The `pointer-events` CSS property sets whether an element triggers pointer events.
```

This isn't quite right- it's whether the element can be the target of pointer events. It would be nice to explain what a pointer is, and also what types of events. Maybe something like this?

```suggestion
description: The `pointer-events` CSS property sets whether a user can interact with an element using a mouse, touch, or other pointing input device.
```